src/canisters/frontend/ic-asset/src/evidence/mod.rs

```diff
@@ -37,11 +40,19 @@ pub async fn compute_evidence(
     let asset_descriptors = gather_asset_descriptors(dirs, logger)?;
 
     let canister_assets = list_assets(canister).await?;
+    let canister_asset_properties = get_assets_properties(canister, &canister_assets)
```

Comment:
Let's add log output before `list_assets` and before and after `get_assets_properties`.  I ran a test locally with 1000 assets and `get_assets_properties` took five seconds, which isn't excessive, but it's long enough that we should give some feedback.

Reply:
thanks for benchmarking this!